src/Pos/Wallet/Web/Doc.hs

```diff
@@ -206,23 +242,61 @@ instance ToSample Coin where
 --
 -- FIXME!
 instance ToSample CHash where
-    toSamples Proxy = error "unmaintained stub for [ToSample:CHash]"
+    toSamples Proxy = singleSample $ CHash "1fi9sA3pRt8bKVibdun57iyWG9VsWZscgQigSik6RHoF5Mv"
```

Comment:
you can probably add to your editor to remove extra witespace at the end of the line